notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb

```diff
@@ -329,32 +323,21 @@
    "source": [
     "## Convert the Model to OpenVINO IR Format [$\\Uparrow$](#Table-of-content:)\n",
     "\n",
-    "Call the OpenVINO Model Optimizer Python API to convert the PaddlePaddle model to OpenVINO IR, with FP32 precision. `mo.convert_model` function accept path to PaddlePaddle model and returns OpenVINO Model class instance which represents this model. Obtained model is ready to use and loading on device using `compile_model` or can be saved on disk using `serialize` function.\n",
-    "See the [Model Optimizer Developer Guide](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) for more information about Model Optimizer."
+    "Call the OpenVINO Model Optimizer Python API to convert the PaddlePaddle model to OpenVINO IR, with FP32 precision. `ov.convert_model` function accept path to PaddlePaddle model and returns OpenVINO Model class instance which represents this model. Obtained model is ready to use and loading on device using `compile_model` or can be saved on disk using `ov.save_model` function.\n",
+    "See the [Model Conversion Guide](https://docs.openvino.ai/2023.0/openvino_docs_model_processing_introduction.html) for more information about Model Optimizer."
```

Comment:
please, remove "about Model Optimizer." from the end

Reply:
removed remaining model optimizer mentions